app/helpers/sticky_newsletter_visibility.rb

```diff
@@ -0,0 +1,15 @@
+module StickyNewsletterVisibility
+  def display_sticky_newsletter_on_this_slug?
+    @slug = request.url.gsub request.base_url, ''
```

Comment:
A personal preference, I would move all the logic of this module to a class, because if you don't the @slug instance variable will be shared with all the controllers, where is possible to cause awkward bugs.

Example:

```
2.1.2 :001 > module Foo
2.1.2 :002?>   def foo
2.1.2 :003?>     @foo = 'foo'
2.1.2 :004?>     end
2.1.2 :005?>   end
 => :foo
2.1.2 :006 > class Bar
2.1.2 :007?>   include Foo
2.1.2 :008?>   end
 => Bar
2.1.2 :009 > bar = Bar.new
 => #<Bar:0x007f99f338fbf0>
2.1.2 :010 > bar.foo
 => "foo"
2.1.2 :011 > bar.instance_variable_get(:@foo)
 => "foo"
```

The @foo variable is using on the current Bar instance. If other method is assigning @slug too, now you will have a bug hard to find.